dts/bindings/i2c/ite,common-i2c.yaml

```diff
@@ -15,7 +15,17 @@ properties:
     port-num:
       type: int
       required: true
+      enum:
+            - 0
+            - 1
+            - 2
+            - 3
+            - 4
+            - 5
       description: Ordinal identifying the port
+        0 = SMB_CHANNEL_A,
+        1 = SMB_CHANNEL_B,
+        2 = SMB_CHANNEL_C,
```

Comment:
What about values greater than 2 (or am I not getting this enum right)?

Reply:
Added the enum of 3~5.